content/en/docs/installation.md

```diff
@@ -331,6 +331,79 @@ flux bootstrap gitlab \
   --path=clusters/my-cluster
 ```
 
+### Bitbucket Server and Data Center
+
+The `bootstrap bitbucket-server` command creates a Bitbucket Server repository if one doesn't exist and
+commits the Flux components manifests to the specified branch. Then it
+configures the target cluster to synchronize with that repository by
+setting up an SSH deploy key or by using token-based authentication.
+
+{{% alert color="info" title="Bitbucket versions" %}}
+This bootstrap command works with Bitbucket Server and Data Center only because it targets the [1.0](https://developer.atlassian.com/server/bitbucket/reference/rest-api/) rest api. Bitbucket Cloud has migrated to the [2.0](https://developer.atlassian.com/cloud/bitbucket/rest/intro/) rest api.
```

Comment:
```suggestion
This bootstrap command works with Bitbucket Server and Data Center only because it targets the [1.0](https://developer.atlassian.com/server/bitbucket/reference/rest-api/) REST API. Bitbucket Cloud has migrated to the [2.0](https://developer.atlassian.com/cloud/bitbucket/rest/intro/) REST API.
```